processor/stream/package_tests/span_attrs_test.go

```diff
@@ -50,7 +49,6 @@ func spanPayloadAttrsNotInFields() *tests.Set {
 	)
 }
 
-// fields in _meta/fields.common.yml that are shared between several data types, but not with spans
 func spanFieldsNotInPayloadAttrs() *tests.Set {
 	return tests.Union(
 		tests.NewSet(
```

Comment:
Since the common fields are not used any more I believe you could remove some of the fields from this exception set (as indicated by the comment you are removing).

Reply:
Correct me if Im wrong, but I think is more like the comment was outdated? All the metadata fields are added to spans now: https://github.com/elastic/apm-server/blob/09951bac1edfb9bb45ba50b2f85081ec988318f3/model/span.go#L206

Reply:
right!